src/westpa/analysis/core.py

```diff
@@ -38,13 +38,13 @@ def open(cls, h5filename='west.h5'):
         Parameters
         ----------
         h5filename : str or file-like object, default 'west.h5'
-            Pathname or stream of a WESTPA HDF5 file.
+            Pathname or stream of a WEST HDF5 file.
```

Comment:
Just to be in line with the terminology of the WESTPA 2.0 paper.

[...] of a main WESTPA HDF5 data file.